playbooks/collect_debug.yml

```diff
@@ -6,6 +6,7 @@
     remote_dir: "/tmp/debug-{{ pipeline_type | default('foreman') }}-{{ pipeline_version | default('nightly') }}-{{ pipeline_os | default('el7') }}"
     smoker_output_dir: '/home/vagrant/smoker'
   roles:
+    - foreman_maintain_packages
```

Comment:
I think we will need some error handling. We could fail gracefully if `foreman-maintain` is not present in the role itself or put a conditional on the roles inclusion. I think I lean towards the former.

Reply:
Maybe include the role if a registered value shows that the foreman-maintain is installed?

Reply:
Added one `when`, the role `foreman_maintain_packages` will only run if the `rubygem-foreman_maintain` is present, for this one pre-task was create to collect the package facts.

Reply:
Should we put that *inside* the role and include it here unconditionally? Feels cleaner as you'd need this safeguard in every pipeline?